symphony-bdk-core/src/main/java/com/symphony/bdk/core/client/ApiClientFactory.java

```diff
@@ -218,14 +231,23 @@ protected void configureProxy(BdkProxyConfig proxyConfig, ApiClientBuilder apiCl
     }
   }
 
-  @API(status = API.Status.INTERNAL)
+
+  /**
+   * @deprecated to be removed if we want to move retry to another module
+   */
+  @Deprecated
+  @API(status = API.Status.DEPRECATED)
```

Comment:
I remember this was to have more meaningful error messages

Reply:
The problem is that is it only used on the HTTP Retry layer, that will soon move to another module. We will then have to break this dependency between `ApiClientFactory` and `RetryWithRecovery` classes